contracts/sentinel/PCVSentinel.sol

```diff
@@ -0,0 +1,180 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
+import "../refs/CoreRef.sol";
+import "../pcv/IPCVDeposit.sol";
+import "./IPCVSentinel.sol";
+import "./IGuard.sol";
+
+/**
+ * @title PCV Sentinel
```

Comment:
**style:** all comments should use `///` and not multi line comments to conform to our codebase

Reply:
Both are natspec and should be acceptable imo